app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,15 @@
+ @import "font-awesome";
+ @import "bootstrap";
+ @import "./config/variables";
+ @import "./components/top-message";
+ @import "./layout/footer";
+ @import "./layout/navbar";
+
+
+
+
+ body{
+  padding: 0!important;
+  margin:0!important;
+  font-family: var(--font-family-main) !important;
+ }
```

Comment:
- Add a space between the class and your curly brace
- remove the `!importants` totally is a bad practice used its.